### ISSUE TITLE
Lower Tx fees

### ISSUE THREAD

**Issue by ioannistsil** [neo-project/neo#2814]
After Ethereum merge to POS NEO became one of most expensive blockchain to invoke smart contracts i believe neo need to lower SC invoke fees..

**Comment by superboyiii** [comment]
0.012 GAS fee for normal NEP-17 transaction = 0.024 USD, can't see any more expensive than Ethereum (0.14 USD for normal ETH transaction, 1.24 USD for ERC20 transaction). Could you show me where Neo is more expensive?

**Comment by ioannistsil** [comment]
Hello @superboyiii  if i want to do a swap will cost me 0.22 GAS 
![image](https://user-images.githubusercontent.com/18231967/191216366-aeb6ce85-3bae-41b3-a2ba-de8044bc520d.png) at current gas price that 52c 
ethereum POS cost 1.25$ 
![image](https://user-images.githubusercontent.com/18231967/191216845-85abc075-5a37-457f-bd10-080310295a97.png)

so if gas price incease to 5$ the cost will be higher than ethereum POS not mention BSC or Matic are far cheaper than NEO SC invoke cost

**Comment by superboyiii** [comment]
> Hello @superboyiii if i want to do a swap will cost me 0.22 GAS ![image](https://user-images.githubusercontent.com/18231967/191216366-aeb6ce85-3bae-41b3-a2ba-de8044bc520d.png) at current gas price that 52c ethereum POS cost 1.25$ ![image](https://user-images.githubusercontent.com/18231967/191216845-85abc075-5a37-457f-bd10-080310295a97.png)
> 
> so if gas price incease to 5$ the cost will be higher than ethereum POS not mention BSC or Matic are far cheaper than NEO SC invoke cost

You system_fee is due to how many syscall is included, I mean how the contract is writing. So maybe you can open an issue on that project to ask them to improve. It's not an issue on Neo network. And if GAS price increase to 5$, ETH will also increase, it's an market behaviour. And in extremely situation, Neo be the top 1 crypto and GAS is very expensive, I think Neo council will vote to reduce the factor of sys_fee since it's better for the whole network.

**Comment by ioannistsil** [comment]
> > Hello @superboyiii if i want to do a swap will cost me 0.22 GAS ![image](https://user-images.githubusercontent.com/18231967/191216366-aeb6ce85-3bae-41b3-a2ba-de8044bc520d.png) at current gas price that 52c ethereum POS cost 1.25$ ![image](https://user-images.githubusercontent.com/18231967/191216845-85abc075-5a37-457f-bd10-080310295a97.png)
> > so if gas price incease to 5$ the cost will be higher than ethereum POS not mention BSC or Matic are far cheaper than NEO SC invoke cost
> 
> Your system_fee is due to how many syscall is included, I mean how the contract is writing. So maybe you can open an issue on that project to ask them to improve. It's not an issue on Neo network. And if GAS price increase to 5$, ETH will also increase, it's a market behaviour. And in extremely situation, Neo becomes the top 1 crypto and GAS is very expensive, I think Neo council will vote to reduce the factor of sys_fee since it's better for the whole network.

Smart contracts will become more and more expensive because we will need to use multi-invoke contracts in the coming future thats the trend of whole market so yes , NEO council should vote to reduce sys fees to stay competive with blockchains like BSC,Polygon , Ethereum POS

**Comment by superboyiii** [comment]
> > > Hello @superboyiii if i want to do a swap will cost me 0.22 GAS ![image](https://user-images.githubusercontent.com/18231967/191216366-aeb6ce85-3bae-41b3-a2ba-de8044bc520d.png) at current gas price that 52c ethereum POS cost 1.25$ ![image](https://user-images.githubusercontent.com/18231967/191216845-85abc075-5a37-457f-bd10-080310295a97.png)
> > > so if gas price incease to 5$ the cost will be higher than ethereum POS not mention BSC or Matic are far cheaper than NEO SC invoke cost
> > 
> > 
> > Your system_fee is due to how many syscall is included, I mean how the contract is writing. So maybe you can open an issue on that project to ask them to improve. It's not an issue on Neo network. And if GAS price increase to 5$, ETH will also increase, it's a market behaviour. And in extremely situation, Neo becomes the top 1 crypto and GAS is very expensive, I think Neo council will vote to reduce the factor of sys_fee since it's better for the whole network.
> 
> Smart contracts will become more and more expensive because we will need to use multi-invoke contracts in the coming future thats the trend of whole market so yes , NEO council should vote to reduce sys fees to stay competive with blockchains like BSC,Polygon , Ethereum POS

Actually, they have done that already. Current sys_fee is 10% of the default.

**Comment by csmuller** [comment]
Hey @ioannistsil, it would be interesting for the developers of the contract you were interacting with to know the high GAS fees that their contracts incur. I believe you used Flamingo Finance, right?

**Comment by ioannistsil** [comment]
> Hey @ioannistsil, it would be interesting for the developers of the contract you were interacting with to know the high GAS fees that their contracts incur. I believe you used Flamingo Finance, right?

yeah thats correnct , we all understand due the complexity of contract and multi invoking functions fees will get higher and higher in coming future so i wanted to point it. Neo council should movitor this kind of issue and adjust invoking fees in order to stay competive with other blockchain regarding invoking fees

**Comment by ediopia** [comment]
[FTW Swap](https://www.forthewin.network/#/swap) can swap with 0.044 gas. SC optimization can lower the fees.
<img width="378" alt="Screen Shot 2022-12-03 at 6 36 21 AM" src="https://user-images.githubusercontent.com/6670368/205446509-caca8983-7b82-41af-ab9d-cfb3261687f5.png">
